.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
     "rules": {
-        "quotes": [2, "single"],
+        "quotes": [1, "single"],
```

Comment:
This is not really part of your story. If you want to change the ESLint rules I would do a separate PR.